ClassPractice/History.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace TicTacToeGame.ClassPractice;
+
+public class History
+{
+    public int XMoveHistory { get; set; }
+
+    public int OMoveHistory { get; set; }
+
+    public DateTime Date { get; set; }
+
+    public string PlayerXName { get; set; }
+
+    public string PlayerOName { get; set; }
+
+    public static History HistoryConstructor(byte[] xMoveHistory, byte[] oMoveHistory, string playerXName, string playerOName)
```

Comment:
remove word HistoryConstructor to make it an actual constructor, and read up on those https://www.w3schools.com/cs/cs_constructors.php